packages/manager/apps/hub/src/components/at-internet/at-internet.constant.js

```diff
@@ -0,0 +1,11 @@
+export default {
```

Comment:
Looks like this can be extracted to a `ovhManagerAtInternet` module 🤔

Reply:
Not sure about those values, copy/paste from `public-cloud`